chia/plotting/plot_tools.py

```diff
@@ -316,3 +316,41 @@ def find_duplicate_plot_IDs(all_filenames=None) -> None:
         for filename_str in duplicate_filenames:
             log_message += "\t" + filename_str + "\n"
         log.warning(f"{log_message}")
+
+
+def load_plot(
+    filename: str,
+) ->  Tuple[PlotInfo,str]:
+    if filename==None:
+        return  {},"use -f to add a plot file, use -h to see usage"
+   
+    filename=Path(filename)
+    if filename.exists():
```

Comment:
you would save one level of indentation in this whole function if you inverted this test, and had an early exit.